### ISSUE TITLE
able to add more "trackers" than calories only

### ISSUE THREAD

**Issue by loicginoux** [loicginoux/joanne_project#62]
In the idea of adding more information to the photo, I was thinking about having personalised trackers.
- at the beginning users could select what sort of thing they want to track "calories", "weight", "lipids", "carbohydrates"...
- in a second phase everyone could create his own personalised tracker.
- when uploading/editing a photo, a tracker will be represented by a input field to fill in (like calories)
- on the Diary, you will be able to select which graph you want to display, calories, weight...

This is for me another idea to make the app more interesting.

**Comment by jdaudier** [comment]
yes, definitely a good idea for future sprints.  I wonder how we should deal with the fact some items are not tied to a photo like weight and not cumulative by the daily meals.

**Comment by loicginoux** [comment]
I set that in sprint 6 but this need to be thought a bit more in details, how it will work, etc... we could define all details during sprint 6 and implement it for sprint 7.... your thoughts?

**Comment by jdaudier** [comment]
sure
